src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,25 @@
+type Props = {
+  errorText: string;
+  setErrorText: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorText,
+  setErrorText,
+}) => {
+  return (
+    <div
+      className="notification is-danger is-light has-text-weight-normal"
+      hidden={!errorText}
+    >
+      <button
+        type="button"
+        className="delete"
+        onClick={() => setErrorText('')}
+      >
+        {}
```

Comment:
Why do you need this empty block?
You can remove it and create `button` as a self-closed tag.